src/makeRobot.js

```diff
@@ -39,6 +39,62 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  return {
+    name: name,
+    'chip version': version,
```

Comment:
```suggestion
    chipVersion: version,
```
Как-то странно выглядит строка с пробелом в качестве ключа